fms2_io/netcdf_io.F90

```diff
@@ -1493,6 +1495,25 @@ function get_valid(fileobj, variable_name) &
       add_offset = 0._real64
     endif
 
+    !Get default max/min from missing_value.  These could be overwritten by
```

Comment:
Change the description to be more explicit; state that max and and min data values are defined by the valid_range, valid_min, and valid_max attributes if they are present. If the fill_value attribute is present and valid_range is not, then fill_value determines valid_data values. Otherwise, the missing_value attribute determines valid data_values if it is present, and valid_range  fill_value attributes are not.

Reply:
Done